src/App.js

```diff
@@ -3,16 +3,24 @@ import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
 import logo from './logo.svg';
 import './App.css';
-
+import LoginWithAmazon from './LoginWithAmazon.js'
 import ChatWindow from './ChatWindow.js'
 
 class App extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
   render() {
     return (
       <MuiThemeProvider>
         <div className="App">
-        
-          <ChatWindow />
+          <LoginWithAmazon setAccessToken={(access_token) => this.setState({access_token: access_token})} />
+
+          {/* This will not be rendered until access_token is available in the state */}
```

Comment:
Add a comment that this is temporary logic. We don't want to control ChatWindow rendering like this in the long term.